### PR TITLE
adding in page nav

### PR DESCRIPTION
**Pull request by hclarkEnq**
## Description

Please include a summary of the changes and any key information a reviewer may need.

## Tickets

* [CNFT1-2886](https://cdc-nbs.atlassian.net/browse/CNFT1-2886)

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-2886]: https://cdc-nbs.atlassian.net/browse/CNFT1-2886?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-ui/src/apps/patient/add/extended/nav/AddPatientExtendedNav.tsx
Comment (mpeels) [inline apps/modernization-ui/src/apps/patient/add/extended/nav/AddPatientExtendedNav.tsx:1]: Issue: Can we move the core functionality of the nav into a generic design system component? Then the `AddPatientExtendedNav` component can simply define the sections and pass them to this new generic component.

**Review by mpeels — CHANGES_REQUESTED** [review]

File: apps/modernization-ui/src/apps/patient/add/extended/nav/AddPatientExtendedNav.tsx
Comment (hclarkEnq) [reply apps/modernization-ui/src/apps/patient/add/extended/nav/AddPatientExtendedNav.tsx:1]: yess

File: apps/modernization-ui/src/apps/patient/add/extended/sections.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    { id: 'name', label: 'Name' },
+    { id: 'address', label: 'Address' },
+    { id: 'phoneAndEmail', label: 'Phone & email' },
+    { id: 'identification', label: 'Identification' },
+    { id: 'race', label: 'Race' },
+    { id: 'ethnicity', label: 'Ethnicity' },
+    { id: 'sexAndBirth', label: 'Sex & birth' },
+    { id: 'mortality', label: 'Mortality' },
+    { id: 'general', label: 'General patient information' }
+];
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/patient/add/extended/sections.ts:17]: Question: Thoughts on moving this inside `AddPatientExtendedNav`?

File: apps/modernization-ui/src/apps/patient/add/extended/sections.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export type Section = {
+    id: string;
+    label: string;
+};
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/patient/add/extended/sections.ts:4]: Issue: Duplicate type for section. Could just use the new `NavSection` defined in `SectionNavigation`

**Review by mpeels — DISMISSED** [review]

**Review by alaapbharadwaj — DISMISSED** [review]

**Pull request closed without merging**